copier.yml

```diff
@@ -352,17 +352,24 @@ cidr_whitelist:
     ⚠️ It must be a list. And this is only supported if you deploy with Traefik 2+.
 
 postgres_version:
-  default: "14"
+  default: "15"
   help: >-
-    Which PostgreSQL version do you want to deploy?
+    Which PostgreSQL version do you want to deploy? (Recommended: for new instances
+    always use the lastest version)
   choices:
     I will use an external PostgreSQL server: ""
-    "9.6": "9.6"
-    "10": "10"
-    "11": "11"
-    "12": "12"
-    "13": "13"
-    "14": "14"
+    "9.6": null
+    "10": null
+    "11": null
+    "12": null
+    "13": null
+    "14": null
+    "15": null
+  validator: >-
+    {% if (odoo_version >= 16.0 and postgres_version < 12) or (odoo_version >= 14.0 and
+    postgres_version < 10) %} The postgres version is too low for the selected Odoo
+    version. {% elif odoo_version <= 12.0 and postgres_version > 14 %} The postgres
+    version is too high for the selected Odoo version. {% endif %}
```

Comment:
This seems like won't work. All values are `null`, but they should be strings, just like before. Also, since they are strings, you need to cast them when doing the version comparison.

Reply:
Umm I still continue without get this to work... :(